import-service/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "remove": "sls remove",
+    "deploy": "sls deploy",
+    "deploy:prod": "sls deploy --stage production"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/pg": "^8.6.5",
+    "aws-sdk": "^2.1202.0",
```

Comment:
version 3 of sdk is already available, you can find info here https://serverless.pub/migrating-to-aws-sdk-v3/